test/raygun.messageBuilder_test.js

```diff
@@ -54,7 +54,46 @@ test("basic builder tests", function (t) {
 
     var message = builder.build();
     tt.ok(message.details.error.message);
-    tt.equal("name=Test", message.details.error.message);
+    tt.equal(message.details.error.message, "name=Test");
+    tt.ok(message.details.groupingKey);
+    tt.end();
+  });
+
+  // Reference: https://github.com/MindscapeHQ/raygun4node/issues/68
+  t.test("humanise Symbol object string", function (tt) {
+    var builder = new MessageBuilder({ useHumanStringForObject: true });
+    builder.setErrorDetails({ symbol: Symbol("Test") });
+    var message = builder.build();
+    tt.ok(message.details.error.message);
+    tt.equal(message.details.error.message, "symbol=Symbol(Test)");
+    tt.ok(message.details.groupingKey);
+    tt.end();
+  });
+
+  t.test("humanise self-referencing complex error string", function (tt) {
+    var builder = new MessageBuilder({ useHumanStringForObject: true });
+
+    // error self-references, causing a potential infinite recursion in the filter method
```

Comment:
Hmmm... I just argued above this should not be allowed. I see you did consider it but plan it as expected behavior.

I think we should not allow such a crash, instead, replace a cyclical object's value with something like `[Circular]`

Reply:
The code already does that. This is a unit test verifying it cannot happen.

The code replaces anything deeper than level 1 with `...` independently if it is a circular reference or not.

All of this works similar to the 3rd party library it replaced.